core/storage/base_model/gae_models.py

```diff
@@ -47,7 +47,7 @@
     # Models that should be pseudonymized in their local context.
     'LOCALLY_PSEUDONYMIZE',
     # Models that should only be kept if published.
```

Comment:
Please update the docstring to match the other noteworthy aspect of this constant: pseudonymization.

Reply:
Done.